### PR TITLE
CODE RUB: Split logic files for Retrieve All Assignments

### PR DESCRIPTION
**Pull request by deepshri7**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/Assignments/AssignmentServiceTests.Logic.cs [hunk 1]
```diff
@@ -15,38 +15,6 @@
...
     public partial class AssignmentServiceTests
-    {        
-        [Fact]
-        public void ShouldRetrieveAllAssignments()
-        {
-            // given
-            DateTimeOffset randomDateTime = GetRandomDateTime();
-            IQueryable<Assignment> randomAssignments = CreateRandomAssignments(randomDateTime);
-            IQueryable<Assignment> storageAssignments = randomAssignments;
-            IQueryable<Assignment> expectedAssignments = storageAssignments;
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/Assignments/AssignmentServiceTests.Logic.cs:26]: Delete the file is it has no more test methods left in it

**Comment by hassanhabib** [comment]
Add `DeepClone` on the `expectedStorageAssignments`

**Comment by hassanhabib** [comment]
@deepshri7 fix conflicts here please.

**Comment by ShriHumrudha** [comment]
Beyond the 3 day window.

**Pull request closed without merging**